packages/utils/src/lib/report.ts

```diff
@@ -161,6 +167,103 @@ export function countCategoryAudits(
   }, 0);
 }
 
+export function getAuditsFromAllPlugins(
+  report: ScoredReport,
+): EnrichedAuditReport[] {
+  return report.plugins.reduce<EnrichedAuditReport[]>(
+    (acc, plugin) => [...acc, ...plugin.audits],
+    [],
+  );
+}
+
+export function getAuditByRef(
+  { slug, weight, plugin }: CategoryRef,
+  plugins: ScoredReport['plugins'],
+): WeighedAuditReport {
+  const auditPlugin = plugins.find(
+    ({ slug }) => slug === plugin,
+  ) as PluginReport;
```

Comment:
It would be better to handle this safely. I see you use optional chaining anyway (`auditPlugin?.`).

```suggestion
  const auditPlugin = plugins.find(
    ({ slug }) => slug === plugin,
  );
```

Reply:
Done